unmappable characters for encoding ASCII fix

```--- RBlock.java	2017-12-19 07:26:41.122165829 +0100
+++ RBlock.java	2017-12-19 07:16:50.885482620 +0100
@@ -1809,10 +1809,10 @@
   }
 
   public static void dumpRndTree(final String indentStr, final boolean isLast, final Renderable r, final boolean condense) {
-    final String nextIndentStr =  indentStr + ((r instanceof RBlockViewport) ? "  " : (isLast ? "  " : "│ "));
-    final String selfIndentStr = (isLast ? "└ " : "├ ");
+    final String nextIndentStr =  indentStr + ((r instanceof RBlockViewport) ? "  " : (isLast ? "  " : "\u2502 "));
+    final String selfIndentStr = (isLast ? "\u2514 " : "\u251C ");
     if (isSimpleLine(r)) {
-      System.out.println(indentStr + selfIndentStr + "─────");
+      System.out.println(indentStr + selfIndentStr + "\u2500\u2500\u2500\u2500\u2500");
     } else {
       if (r instanceof RBlockViewport) {
         // System.out.println(indentStr + "^RBV");
```

jdk8-openjdk 8.u144-1
apache-ant 1.9.7-1

```
build-project:
     [echo] HTML_Renderer: /home/cy/git/git/gngr/src/HTML_Renderer/build.xml
    [javac] Compiling 1 source file to /home/cy/git/git/gngr/src/HTML_Renderer/bin
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.8
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1812: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]     final String nextIndentStr =  indentStr + ((r instanceof RBlockViewport) ? "  " : (isLast ? "  " : "??? "));
    [javac]                                                                                                         ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1812: error: unmappable character (0x94) for encoding US-ASCII
    [javac]     final String nextIndentStr =  indentStr + ((r instanceof RBlockViewport) ? "  " : (isLast ? "  " : "??? "));
    [javac]                                                                                                          ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1812: error: unmappable character (0x82) for encoding US-ASCII
    [javac]     final String nextIndentStr =  indentStr + ((r instanceof RBlockViewport) ? "  " : (isLast ? "  " : "??? "));
    [javac]                                                                                                           ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1813: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]     final String selfIndentStr = (isLast ? "??? " : "??? ");
    [javac]                                             ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1813: error: unmappable character (0x94) for encoding US-ASCII
    [javac]     final String selfIndentStr = (isLast ? "??? " : "??? ");
    [javac]                                              ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1813: error: unmappable character (0x94) for encoding US-ASCII
    [javac]     final String selfIndentStr = (isLast ? "??? " : "??? ");
    [javac]                                               ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1813: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]     final String selfIndentStr = (isLast ? "??? " : "??? ");
    [javac]                                                      ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1813: error: unmappable character (0x94) for encoding US-ASCII
    [javac]     final String selfIndentStr = (isLast ? "??? " : "??? ");
    [javac]                                                       ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1813: error: unmappable character (0x9C) for encoding US-ASCII
    [javac]     final String selfIndentStr = (isLast ? "??? " : "??? ");
    [javac]                                                        ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                       ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x94) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                        ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x80) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                         ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                          ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x94) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                           ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x80) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                            ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                             ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x94) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                              ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x80) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                               ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                                ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x94) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                                 ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x80) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                                  ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0xE2) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                                   ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x94) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]                                                                    ^
    [javac] /home/cy/git/git/gngr/src/HTML_Renderer/org/lobobrowser/html/renderer/RBlock.java:1815: error: unmappable character (0x80) for encoding US-ASCII
    [javac]       System.out.println(indentStr + selfIndentStr + "???????????????");
    [javac]  
```

--- a/src/Platform_Core/build.xml
+++ b/src/Platform_Core/build.xml
@@ -73,7 +73,7 @@
     </target>
     <target name="EntryPoint">
         <java classname="org.lobobrowser.main.EntryPoint" failonerror="true" fork="yes">
-            <jvmarg line="-Dext.dirs=../XAMJ_Build/ext  -Dext.files=../HTML_Renderer/bin,../Primary_Extension/bin  -Djava.security.debugxx=accesss,failure -Djava.security.debugxx=access,domain,stack"/>
+            <jvmarg line="-Dfile.encoding=utf-8 -Dext.dirs=../XAMJ_Build/ext  -Dext.files=../HTML_Renderer/bin,../Primary_Extension/bin  -Djava.security.debugxx=accesss,failure -Djava.security.debugxx=access,domain,stack"/>
             <arg value="-debug"/>
             <arg value="-grinder-key=${gngr.grinder.key}" if:set="gngr.grinder.key"/>
             <classpath refid="Platform_Core.classpath"/>

